README.md

```diff
@@ -402,3 +404,5 @@ Checking out the company's Tech stack through [Stackshare](https://stackshare.io
 | Company | Description | Locations |
 | :------ | :---------- | :-------- |
 | [Avenue](https://avenue.place) [:rocket:](https://careers.avenue.place) | A communications and collaboration platform for DAOs. | `Porto` `Remote` |
+| [Achorage Digital](https://www.anchorage.com) [:rocket:](https://www.anchorage.com/careers) | Trusted institutional partner in crypto and first federally chartered crypto bank. | `Porto` `Remote` |
```

Comment:
Anchorage is already listed, please update the existing one or remove this change.

Reply:
removed